src/content_scripts/profilePage.js

```diff
@@ -22,13 +22,16 @@ function createFollowButton() {
   username = username.split('#')[0];
   username = username.split('?')[0];
 
-  $(document).ready(function () {
-      if(window.location.href.indexOf("profile") > -1) {
-        $followButton = $('<button></button>').attr({ id: 'se_follow_button' }).css("margin-top", "6px");
-      } else {
-        $followButton = $('<button></button>').attr({ id: 'se_follow_button' }).css("bottom", "5px");
-      }
-  });
+  function makeStyledButton(profileElement, profileValue, generalElement, generalValue) {
+    if($('#tabs').find("a:first").hasClass("youarehere")) {
```

Comment:
We are relying on implementation details of the site which is likely to change sooner than later. I understand URL check is also not a robust way of doing this but still better than this.

Moreover, class names like `yourarehere` looks like a developer has left this class name unintentionally in the code and is far away from the standard classes being used.

I'd rather go with profile url check than this selector technique.